src/containers/catalog/CatalogReducer.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+import * as actionTypes from './CatalogActionTypes';
+
+export const INITIAL_STATE = {
+  asyncState: {
+    isLoading: true,
+    errorMessage: ""
```

Comment:
please use single quotes. `eslint` should yell at you if you do `npm run lint:js`